IdentityCore/src/broker_operation/request/account_request/MSIDBrokerOperationGetAccountsRequest.m

```diff
@@ -73,6 +79,7 @@ - (NSDictionary *)jsonDictionary
     json[MSID_BROKER_CLIENT_ID_KEY] = self.clientId;
     
     json[MSID_BROKER_FAMILY_ID_KEY] = self.familyId;
+    json[MSID_BROKER_SIGNED_IN_ACCOUNTS_ONLY_KEY] = @(self.returnOnlySignedInAccounts);
```

Comment:
It is better to use "string value", like here:

`json[MSID_BROKER_OPERATION_RESULT_JSON_KEY] = [@(self.success) stringValue];`